File: DataFormManagerApp/src/app/model/form-data-model.ts
import { DataValueModel } from './data-value-model';

export class FormDataModel {
    FormId: number;
    FormType: string;
    FormData: Array<DataValueModel>;
    constructor(formType: string, formData: Array<DataValueModel>, formId: number = null) {
        this.FormId = formId;
        this.FormType = formType;
        this.FormData = formData;
    }
}

File: DataFormManager.Models/FormDataModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DataFormManager.Models
{
    public class FormDataModel
    {
        public string FormType
        {
            get;
            set;
        }
        public List<DataValueModel> FormData
        {
            get;
            set;
        }

    }
}
